### PR TITLE
WIP: 435 help search

### PR DESCRIPTION
**Pull request by dominoxp**

### CODE REVIEW

File: frontend/src/Server/BlazorBoilerplate.Server/Startup.cs [hunk 1]
```diff
@@ -81,6 +81,8 @@ public Startup(IConfiguration configuration, IWebHostEnvironment env)
         }
         public void ConfigureServices(IServiceCollection services)
         {
+            services.UseGTour();
```
Comment (AlexanderZender) [inline frontend/src/Server/BlazorBoilerplate.Server/Startup.cs:84]: This is already at line 182

File: frontend/src/Shared/Modules/BlazorBoilerplate.Theme.MudBlazor.Demo/Shared/Components/UserTour.razor [hunk 2]
```diff
@@ -339,6 +339,8 @@
 </style>
 
 @code {
+    [Inject]
+    private GTour.Abstractions.IGTourService GTourService { get; set; }
```
Comment (AlexanderZender) [inline frontend/src/Shared/Modules/BlazorBoilerplate.Theme.MudBlazor.Demo/Shared/Components/UserTour.razor:343]: Is this necessary?

File: frontend/src/Shared/Modules/BlazorBoilerplate.Theme.MudBlazor/BlazorBoilerplate.Theme.MudBlazor.csproj [hunk 3]
```diff
@@ -10,12 +10,13 @@
...
     <PackageReference Include="BytexDigital.Blazor.Components.CookieConsent" Version="1.0.15" />
     <PackageReference Include="CXuesong.MW.WikiClientLibrary" Version="0.7.5" />
-    <PackageReference Include="MudBlazor" Version="6.11.2" />
+    <PackageReference Include="MudBlazor" Version="6.12.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
     <PackageReference Include="System.Net.Http.Json" Version="6.0.0" />
     <PackageReference Include="Toolbelt.Blazor.LoadingBar" Version="13.1.2" />
     <PackageReference Include="Wiki.Net" Version="4.1.0" />
+    <PackageReference Include="STGTour.GTour" Version="1.0.8" />
```
Comment (AlexanderZender) [inline frontend/src/Shared/Modules/BlazorBoilerplate.Theme.MudBlazor/BlazorBoilerplate.Theme.MudBlazor.csproj:19]: This shouldnt be here as we use a custom version

**Review by AlexanderZender — COMMENTED** [review]
Fix and check comments

**Comment by AlexanderZender** [comment]
Is there a reason on the training overview page the metric selectbox is locked even after results are available?

**Comment by AlexanderZender** [comment]
image

**Review by AlexanderZender — APPROVED** [review]

**Comment by Granoffel** [comment]
> image

The metric drop-down should now work. 

- [ ] If still loading (no metric) field is disabled. 

- [ ] If training finished (metrics loaded) drop down is selectable.

**Review by AlexanderZender — APPROVED** [review]

**Pull request merged**